src/main/kotlin/io/github/gradlenexus/publishplugin/NexusRepository.kt

```diff
@@ -48,6 +52,18 @@ open class NexusRepository @Inject constructor(@Input val name: String, project:
     @Input
     val stagingProfileId = project.objects.property<String>()
 
+    @Nested
+    val retrying = project.objects.property<RetryingConfig>().apply {
+        set(RetryingConfig(project))
+    }
+
+    fun retrying(action: Action<in RetryingConfig>) = action.execute(retrying.get())
+
+    //TODO: Fails anyway with Groovy for "maxNumber = 10" with "Cannot cast object '10' with class 'java.lang.Integer' to class 'org.gradle.api.provider.Property'"
+    fun retrying(closure: Closure<*>) {
```

Comment:
:x: We don't need this method because you can write an `Action` as a closure in Groovy.

Reply:
I believe that there was something mentioned in the Gradle documentation that Gradle doesn't support the second level of nesting domain objects and it is needed. Unfortunately, I cannot find it right now. Nevertheless, it doesn't work as mentioned in the comment for Groovy anyway, so I maybe remove it anyway.